### DISCUSSION TITLE
A few questions and problem

### DISCUSSION THREAD

**Discussion by snipecoder** [postalserver/postal#2997] [Help with using Postal]
Hello

I just found out about Postal, which is great. I have ZoneMTA - Is it possible to connect ZoneMTA to the Postal as a delivery server?

Does Postal have a feature for holding back emails to destinations that have previously hard-bounced?

Also, how do I set Postal to listen to multiple ports like 25/587 and 465?

I hope you can help me, 
Thank you.

**Comment by willpower232** [comment]
> Is it possible to connect ZoneMTA to the Postal as a delivery server?

Technically it is possible but it was not what Postal was intended for so you will probably find that it doesn't work out that well. Postal mostly allows software to send emails.

> Does Postal have a feature for holding back emails to destinations that have previously hard-bounced?

Postal will block sending in various failure conditions if an address ends up on the suppression list.

> how do I set Postal to listen to multiple ports like 25/587 and 465?

You would need to use iptables port forwarding or similar to pass the traffic around however Postal does not do implicit TLS, only STARTTLS, so even if you make some ports available, it is likely you will struggle on some of them.
